### ISSUE TITLE
Typing outside of an input doesn't autofocus the message field

### ISSUE THREAD

**Issue by schrottkatze** [ajbura/cinny#1579]
### Describe the bug

See the title. 

I foudn this: https://github.com/cinnyapp/cinny/issues/112

Which apparently led to this being added 2 years ago, but apparently that doesn't work anymore.

### Reproduction

1. open a chat
2. don't be autofocused on a message field
3. try to type

### Expected behavior

being able to type automatically inside of the message field

### Platform and versions

```shell
1. OS: NixOS 24.05.20231209.666fc80 (Uakari) x86_6
2. Browser: N/A
3. Cinny desktop v3.1.0
4. katzen.cafe
```

**Comment by jacks0n9** [comment]
Yeah, I came here to suggest this. I second this issue.

**Comment by ajbura** [comment]
this has been fixed in v4.0.0

**Comment by jacks0n9** [comment]
thank you
